contracts/plugins/assets/curve/CurveStableMetapoolCollateral.sol

```diff
@@ -122,7 +122,8 @@ contract CurveStableMetapoolCollateral is CurveStableCollateral {
         high = aumHigh.div(supply, CEIL);
         assert(low <= high); // not obviously true just by inspection
 
-        return (low, high, 0);
+        pegPrice = 0; // can't deduce from MEV-manipulable pricing unfortunately
```

Comment:
We should make it clear that 0 is a special state where the BasketHandler will skip adding the issuance premium if the value is 0.